frontend/integration-tests/tests/kubevirt/kubevirt.login.scenario.ts

```diff
@@ -0,0 +1,14 @@
+import { browser } from 'protractor';
+import { appHost } from '../../protractor.conf';
+import { execSync } from 'child_process';
+import { logIn } from './utils/utils';
+
+describe('Authentication', () => {
+  it('Web console logs in.', async() => {
+    await browser.get(appHost);
+    if (process.env.BRIDGE_BASE_ADDRESS !== undefined) {
+      await logIn();
+      execSync(`oc login -u ${process.env.BRIDGE_AUTH_USERNAME} -p ${process.env.BRIDGE_AUTH_PASSWORD} --config=${process.env.KUBECONFIG}`);
```

Comment:
`config=${process.env.KUBECONFIG}` should be unnecessary. `oc` will already read the env var.